unlock-app/src/components/interface/checkout/main/Password.tsx

```diff
@@ -79,6 +88,20 @@ export function Password({ injectedProvider, checkoutService }: Props) {
             {...register('password', {
               required: true,
               min: 1,
+              validate: (password: string) => {
+                const encoded = ethers.utils.defaultAbiCoder.encode(
```

Comment:
Let's move this logic to helper/utils since we already have it in multiple places.

Reply:
`getEthersWalletFromPassword`